s3-source-connector/src/integration-test/java/io/aiven/kafka/connect/s3/source/IntegrationTest.java

```diff
@@ -195,11 +206,11 @@ void avroTest(final TestInfo testInfo) throws IOException, InterruptedException
         final Schema.Parser parser = new Schema.Parser();
         final Schema schema = parser.parse(schemaJson);
 
-        final byte[] outputStream1 = getAvroRecord(schema, 1, 100);
-        final byte[] outputStream2 = getAvroRecord(schema, 101, 100);
-        final byte[] outputStream3 = getAvroRecord(schema, 201, 100);
-        final byte[] outputStream4 = getAvroRecord(schema, 301, 100);
-        final byte[] outputStream5 = getAvroRecord(schema, 401, 100);
+        final byte[] outputStream1 = generateNextAvroMessagesStartingFromId(1, 100, schema);
```

Comment:
Instead of hardcoding as 100, can we introduce a variable, and apply on top of it ? Like n+1 or 2n+1 .. 
so it would be convenient to test large messages/records in future ?

Reply:
Let's track this in KCON-63!